wled00/json.cpp

```diff
@@ -279,7 +279,7 @@ bool deserializeState(JsonObject root)
 
   JsonObject playlist = root[F("playlist")];
   if (!playlist.isNull()) {
-    loadPlaylist(playlist); return stateResponse;
+    loadPlaylist(playlist); //return stateResponse;
```

Comment:
is this change intentional? if so: what does it do?

Reply:
It is, otherwise WLED is not notified by the playlist change and behaves "strange".
For example: if "on":false is specified in json, the power button changes to off but the effect keeps playing.